### DISCUSSION TITLE
Welcome to keycloakify-tailwind-shadcn Discussions!

### DISCUSSION THREAD

**Discussion by nima70** [nima70/keycloakify-tailwind-shadcn#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anshulgoel27** [comment]
Any update on supporting the latest version of keycloak?

**Comment by nima70** [comment]
Hi @anshulgoel27 ,
No one has requested that before, but since you're interested, I'll give it a shot next weekend. Stay tuned!
